src/briefcase/commands/create.py

```diff
@@ -292,16 +292,19 @@ def _unpack_support_package(self, support_file_path, support_path):
         except (shutil.ReadError, EOFError) as e:
             raise InvalidSupportPackage(support_file_path) from e
 
-    def install_app_support_package(self, app: BaseConfig):
+    def install_app_support_package(self, app: BaseConfig, update: bool = False):
```

Comment:
I can see why you've added this boolean flag; however, in practice, I don't think it's needed. If we simplify the logic so that we *always* delete the support package if it always exists, that logic won't ever trigger for a clean build, and will still clean up on an update.